inception-project/inception
# #2312 - Add support for new external recommender v2

**What's in the PR**
- [x] Move old external recommender to v1 subpackage
- [x] Can check connection to v2 recommender
- [x] Can select classifiers from the server
- [x] Predicting works
- [ ] Training works

**How to test manually**
* Check out https://github.com/inception-project/inception-external-recommender-v2
* Install with test dependencies `pip install -e ".[all]"`
* Run `make inception_test`
* Try setting up a v2 recommender using this service

**Automatic testing**

**Documentation**
